apps/web/components/ui/PurchaseCard/PurchaseCard.vue

```diff
@@ -196,10 +196,7 @@ const handleAddToCart = async (quickCheckout = true) => {
   };
 
   const added = await addToCart(params);
-  if (added) {
-    if (quickCheckout) openQuickCheckout(product, quantitySelectorValue.value);
-    send({ message: t('addedToCart'), type: 'positive' });
-  }
+  if (added && quickCheckout) openQuickCheckout(product, quantitySelectorValue.value);
```

Comment:
Please consider that even if the quick checkout visibility is not configured in the environment, this method still includes the `quickCheckout` parameter with a default value of `true`. This means clients have the option to disable it. In scenarios where it is disabled by the client, we need to ensure that the notification is displayed.